lib/scoped_search/query_builder.rb

```diff
@@ -485,6 +489,16 @@ def to_sql(builder, definition, &block)
             raise ScopedSearch::QueryNotSupported, "Don't know how to handle this operator node: #{operator.inspect} with #{children.inspect}!"
           end
         end
+
+        private
+
+        def validate_value(field, value)
+          validator = field.validator
+          if field.validator && field.only_explicit
```

Comment:
Why check `only_explicit` here? If you're searching with a field name then the validation won't fire unless this option's set. I don't _think_ validator should require only_explicit.